flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -995,6 +995,56 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     assertEquals(false, tableSchema2.getPrimaryKey.isPresent)
   }
 
+  @Test
+  def testAlterTableWithIfExists(): Unit = {
```

Comment:
I think a whole separate IT case for this isn't necessary, this is really quite heavy on the test performance.